github-actions/scan/src/input-helper.ts

```diff
@@ -21,6 +22,27 @@ export function split(input: string): string[] {
         .filter(item => item.length > 0);
 }
 
+export function ensureAcceptedScanType(data: string[]): string[] {
```

Comment:
I would prefer a name change here - makes it more clear that this method returns types here.

```suggestion
export function safeAcceptedScanTypes(data: string[]): string[] {
```